app/controllers/api/properties_controller.rb

```diff
@@ -1,5 +1,6 @@
 module Api
   class PropertiesController < Api::ApiController
+
```

Comment:
Not a big deal, but try to avoid white space and other meaningless changes with your commits. They become noise later if we need to search the history.